src/java/com/unifina/feed/ICatchupFeed.java

```diff
@@ -11,15 +11,6 @@
  * the normal event queue.
  */
 public interface ICatchupFeed {
```

Comment:
This interface could be removed, I think? (related to removed catchup feature)

Also its implementation in `AbstractFeedProxy` is then unnecessary.